streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsRebalanceListener.java

```diff
@@ -97,16 +91,10 @@ public void onPartitionsLost(final Collection<TopicPartition> partitions) {
         try {
             // close all active tasks as lost but don't try to commit offsets as we no longer own them
             taskManager.handleLostAll();
-        } catch (final Throwable t) {
-            log.error(
-                "Error caught during partitions lost, " +
-                    "will abort the current process and re-throw at the end of rebalance: ",
-                t
-            );
-            streamThread.setRebalanceException(t);
+        } catch (final Exception e) {
```

Comment:
Hmm.. do we want to capture all Exceptions as TaskMigratedException? I actually think it's okay to not capture anything here since only StreamsException or TaskMigratedException would be thrown here.

Reply:
Sounds good